server/routes/auth.js

```diff
@@ -17,8 +17,7 @@ const createAuthRouter = ({ signIn, signInCallback, signOut }) => {
 
   router.get('/error', (req, res) => {
     config.detailsType = 'small';
-    config.userName = req.user && req.user.getFullName();
-
+    res.locals.userName = req.user && req.user.getFullName();
```

Comment:
this gets called before new middleware, the template has changed to read this from res.locals